website/pages/docs/agent/options.mdx

```diff
@@ -955,6 +955,22 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
 
 - `bind_addr` Equivalent to the [`-bind` command-line flag](#_bind).
 
+- `cache` Cache configuration of agent. The configurable values are the following:
+
+  - `rate_limit_burst`: The size of the token bucket used to recharge the rate-limit per
+    cache entry. The default value is 2 and means that when cache has not been updated
+    for a long time, 2 successive queries can be made as long as the rate-limit is not
+    reached.
+
+  - `rate_limit_per_entry`: configures the rate-limit at which the cache may refresh a single
```

Comment:
```suggestion
  - `entry_fetch_rate_limit`: configures the rate-limit at which the cache may refresh a single
```

Reply:
ok, I'll change that tomorrow

Reply:
Fixed in https://github.com/hashicorp/consul/pull/8226/commits/00df40b9f5a5f15d0e55095c06b0a007bd74db6e